lib/engines/hcktest/drivers/vioscsi.json

```diff
@@ -13,6 +13,22 @@
     {
       "desc": "Make Disk Writable",
       "run": "Set-Disk -Number 1 -IsReadonly $False"
+    },
+    {
+      "desc": "Initialize Disk",
+      "run": "Initialize-Disk -Number 1 -PartitionStyle GPT"
+    },
+    {
+      "desc": "Remove Existing Partitions",
+      "run": "Get-Partition -DiskNumber 1 | Remove-Partition -Confirm:$false"
```

Comment:
It looks like `Remove-Partition` can directly take `-DiskNumber` and `-PartitionNumber`. It's already supported by Windows 2012: https://learn.microsoft.com/en-us/powershell/module/storage/remove-partition?view=winserver2012-ps